src/server/index.js

```diff
@@ -29,6 +32,7 @@ app.use(
 
 app.use(authRouter);
 app.use('/media', s3Router);
+app.use(express.static('../../dist'));
```

Comment:
@gnarf @ktbee Is the app going to be server side rendered? If so, routing doesn't work right now.

Reply:
It needs that "default 404 page is index.html" thing to work

Re: SSR - doubtful - we might do it as a feature longer term, but I would almost see this "server" as having nothing to do with the frontend... I.E. not even doing this express.static longer term...

The long term goal IMO:  This is an "API" server, there is a "frontend" and the "api" server and they are just mounted at different directories on the "front server"...

I.E. lets let nginx / etc do the html / dist serving and then proxy requests to /api/whatever to the api server.

As a "yarn start" frontend, you can use a proxy on the frontend's webpack to proxy to the api server for /api routes.


All this "longer term" stuff doesn't need to happen >now< but you should have its concepts ready to go sooner or later.